### PR TITLE
Cargo workspace

### PR DESCRIPTION
**Pull request by sporksmith**

### CODE REVIEW

**Comment by sporksmith** [comment]
Sorry for the post-review-request noise. I *think* it's actually ready now

**Review by stevenengler — APPROVED** [review]

File: src/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[workspace]
+
+members = [
+    "main",
+    "test",
+    "lib/gml-parser",
+    "lib/logger/rust_bindings",
+    "lib/syscall-logger",
+]
```
[the remark this answers is not in the document]
Comment (sporksmith) [reply src/Cargo.toml:9]: Done. Is there some way to test? Hopefully we'll see some kind of error if we merge and dependabot doesn't like t hew new config?

[the remark this answers is not in the document]
Comment (stevenengler) [reply src/Cargo.toml:9]: I think only Rob can see the dependabot status after it's merged. I'm not sure what will happen with the existing PRs though.

[the remark this answers is not in the document]
Comment (sporksmith) [reply src/Cargo.toml:9]: Ok. I'll just go ahead and merge and we'll see what happens :)

**Comment by robgjansen** [comment]
`rust-analyzer` is looking for a `Cargo.toml` in my project root. How do I tell it to look for the new workspace one in `src` instead? (I used to do this by dropping a Cargo workspace file in my root, but now it doesn't want me to have 2 of those.)

**Comment by robgjansen** [comment]
Oh, nevermind, restarting VSCode several times fixed it 🗡️

**Pull request merged**